loki/transform/transform_dead_code.py

```diff
@@ -63,4 +64,5 @@ def visit_Conditional(self, o, **kwargs):
         if condition == 'False':
             return else_body
 
-        return self._rebuild(o, tuple((condition,) + (body,) + (else_body,)))
+        has_elseif = isinstance(else_body, Conditional)
```

Comment:
Hm, not sure this works as intended. The `else_body` should always be a tuple, not an individual node - and it might misclassify genuinely nested (albeit equivalent) conditionals as `has_elseif`.
I think something like this might work:
```suggestion
        has_elseif = o.has_elseif and else_body and isinstance(else_body[0], Conditional)
```

Reply:
Gotcha! Many thanks, fixed now (with additional testing).